pkg/textparse/parse.go

```diff
@@ -237,6 +237,7 @@ const (
 type MetricType string
 
 const (
+	MetricTypeUnknown   = ""
```

Comment:
This should have untyped as a value (or unknown if you want to future-proof for openmetrics).

Reply:
Yes, missed untyped intitially but have this added locally as yet another one. Untyped is known and unknown. I wanted `MetricTypeUnknown` for unknown unknown.

Reply:
Within the current format untyped is that, there is no "unknown".

Reply:
Fair, fine with using a boolean in places where that doesn't quite cut it.

Reply:
What cases are you thinking of?

Reply:
With the new changes it's no longer relevant I suppose.